tests/backends/sqlite/tests.py

```diff
@@ -162,3 +162,27 @@ def create_object():
         thread.start()
         thread.join()
         self.assertEqual(Object.objects.count(), 2)
+
+
+@unittest.skipUnless(connection.vendor == 'sqlite', 'SQLite tests')
+class IntrospectionTests(TestCase):
```

Comment:
Please use a separate `test_introspection.py` file.

Reply:
I wasn't sure at the time, thanks for the clarification.